model_bakery/utils.py

```diff
@@ -45,5 +45,15 @@ def seq(value, increment_by=1, start=None):
             else:
                 yield series_date
     else:
+        if suffix and type(suffix) is not str:
```

Comment:
```suggestion
        if suffix and not isinstance(suffix, str):
```

Nice explanation of why [here](https://stackoverflow.com/a/1549854).

Reply:
Totally agree. I was simply going for consistency with the rest of the function.

I will update this and other usage as well.